### ISSUE TITLE
Error loading connector/postgres@1.0.2. pg is not defined when not using connection pooling

### ISSUE THREAD

**Issue by maximilianoPizarro** [Axway-API-Builder-Ext/api-builder-extras#69]
**Artefact version you are using @axway-api-builder-ext/api-builder-plugin-dc-postgres: ^1.0.2**

**API-Builder version you are using 1.18.1**

**Actual behavior**
```
module.exports = {
	connectors: {
		postgres: {
			connector: '@axway-api-builder-ext/api-builder-plugin-dc-postgres',
			host: process.env.POSTGRES_HOST,
			port: process.env.POSTGRES_PORT,
			database: process.env.POSTGRES_DB,
			user: process.env.POSTGRES_USER,
			password: process.env.POSTGRES_PASSWORD,

			// Create models based on your schema that can be used in your API.
			generateModelsFromSchema: true,

			// Whether or not to generate APIs based on the methods in generated models.
			modelAutogen: true
		}
	}
};
```

**Expected behavior**

```
connect to postgresql 13.3
```

**Log-Output in Debug**

```
ERROR Error loading connector/postgres@1.0.2. pg is not defined
ERROR Server failed to load components
ERROR ReferenceError: pg is not defined
    at Connector.exports.connect [as _connect] (C:\Users\Max\demo-posgresql-axway\demo-posgresql-axway\node_modules\@axway-api-builder-ext\api-builder-plugin-dc-postgres\lib\lifecycle\connect.js:24:21)
    at Connector.connectTask (C:\Users\Max\demo-posgresql-axway\demo-posgresql-axway\node_modules\arrow-orm\lib\connector.js:478:9)
    at C:\Users\Max\demo-posgresql-axway\demo-posgresql-axway\node_modules\async\dist\async.js:3880:24
    at replenish (C:\Users\Max\demo-posgresql-axway\demo-posgresql-axway\node_modules\async\dist\async.js:1011:17)
    at C:\Users\Max\demo-posgresql-axway\demo-posgresql-axway\node_modules\async\dist\async.js:1016:9
    at eachOfLimit (C:\Users\Max\demo-posgresql-axway\demo-posgresql-axway\node_modules\async\dist\async.js:1041:24)
    at C:\Users\Max\demo-posgresql-axway\demo-posgresql-axway\node_modules\async\dist\async.js:1046:16
    at _parallel (C:\Users\Max\demo-posgresql-axway\demo-posgresql-axway\node_modules\async\dist\async.js:3879:5)
    at Object.series (C:\Users\Max\demo-posgresql-axway\demo-posgresql-axway\node_modules\async\dist\async.js:4735:5)
    at Connector.connect (C:\Users\Max\demo-posgresql-axway\demo-posgresql-axway\node_modules\arrow-orm\lib\connector.js:517:8)
```

**Comment by cwiechmann** [comment]
Thanks for reporting this issue @maximilianoPizarro. It's clearly a bug and it will be fixed.

In the meantime, can you please the following parameters into the configuration:
```
connectionPooling: true,
connectionLimit: 10,
```

This should fix the behavior as you see it. An upcoming fix very likely sets these settings as default and fixes the missing property error.

**Comment by cwiechmann** [comment]
The issue has been fixed with version 1.0.3.
